src/UserSpaceInstrumentation/TestUtils.h

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef USER_SPACE_INSTRUMENTATION_DUMP_DISASSEMBLY_H_
+#define USER_SPACE_INSTRUMENTATION_DUMP_DISASSEMBLY_H_
+
+#include <cstdint>
+#include <string_view>
+#include <vector>
+
+#include "AddressRange.h"
+
+namespace orbit_user_space_instrumentation {
+
+// Returns the file offset of function `function_name` in the test executable.
```

Comment:
From the implementation I understand that this is the relative address, not the "offset" (which is relative address minus load bias).

To distinguish from the other function, this can be called
`GetFunctionRelativeAddressRangeOrDie` (I would drop "in file", the concept of relative address still applies in memory).

@dimitry- might have an opinion backed by better knowledge.

Reply:
yep, looks like this are virtual addresses not offsets.